pyvisa/cmd_line_tools.py

```diff
@@ -43,6 +51,11 @@ def visa_main(command: Optional[str] = None) -> None:
         subparsers.add_parser("shell", help="start the PyVISA console")
 
     args = parser.parse_args()
+    logging.basicConfig(
+        level={0: logging.ERROR, 1: logging.WARN, 2: logging.INFO, 3: logging.DEBUG}[
```

Comment:
any reason for an int-indexed dict instead of a tuple/list?

Reply:
None at all! A tuple or list would be more compact, no less explicit, & works just as well.

Reply:
Can you address this then ?